charts/linkerd2/templates/_config.tpl

```diff
@@ -66,8 +66,15 @@
 {{- end -}}
 
 {{- define "linkerd.configs.install" -}}
+{{- if ge (.Values.controllerReplicas | int) 3 -}}
+{
+  "cliVersion":"{{ .Values.global.linkerdVersion }}",
+  "flags":[{"name":"ha","value":"true"}]
```

Comment:
I could be missing something here, but can we add a if case for only the `flags` field?

Reply:
You are right, I corrected it.